motion/motion_launch/launch/motion.launch

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<launch>
+
+  <env name="ROSCONSOLE_FORMAT" value="[${severity}] [${time}] [${node}]: ${message}" />
```

Comment:
This is already in the top-level launch file

Reply:
Addressed in aa8b3c5